tests/schema_test.go

```diff
@@ -15,34 +15,50 @@ func TestSchemaSuite(t *testing.T) {
 	suite.Run(t, new(SchemaSuite))
 }
 
+func (s *SchemaSuite) TestSchemaID() {
+	s.Equal("id", Schema.SchemaFixture.ID.String())
+}
+
 func (s *SchemaSuite) TestSchemaBasicField() {
-	s.Equal("string", Schema.SchemaFixture.String)
+	s.Equal("string", Schema.SchemaFixture.String.String())
 }
 
 func (s *SchemaSuite) TestSchemaRanamedField() {
-	s.Equal("int", Schema.SchemaFixture.Int)
+	s.Equal("int", Schema.SchemaFixture.Int.String())
 }
 
+// TODO: https://github.com/src-d/go-kallax/issues/43
```

Comment:
Same as above; in this case,  `s.Equal("inline", Schema.SchemaFixture.Inline.String())` does not compile because `Schema.SchemaFixture.Inline` does not exist.

Or did I misunderstood you?